app/proposals_whitelisting.go

```diff
@@ -19,21 +19,14 @@ func IsConsumerProposalWhitelisted(content govtypes.Content) bool {
 	switch c := content.(type) {
 	case *proposal.ParameterChangeProposal:
 		return isConsumerParamChangeWhitelisted(c.Changes)
-	case *govtypes.TextProposal,
-		*upgradetypes.SoftwareUpgradeProposal,
+	case *upgradetypes.SoftwareUpgradeProposal,
```

Comment:
i have a question regarding the remaining proposal types. here (https://github.com/neutron-org/neutron/blob/56875ea0bd6765ad4504778a1f0808164504b11e/app/app.go#L464) we register some of the proposal types while setting up the `wasmd` module. do we need to register the remaining proposal types there?

Reply:
This one was updated in PR. I just moved `wasmd`-related `Route` initialization after creating `wasmKeeper` init